.github/workflows/artifacts.yml

```diff
@@ -0,0 +1,21 @@
+name: Comment Artifacts
+on:
+  workflow_run:
+    workflows: [Build Docs]
```

Comment:
does this need to match the name `docs` rather than `Build Docs`?

Reply:
According to the [documentation](https://docs.github.com/en/github-ae@latest/actions/using-workflows/events-that-trigger-workflows#workflow_run), it should be the name (`Build docs`), not the workflow filename. 

Also, per the docs:

> Note: This event will only trigger a workflow run if the workflow file is on the default branch.

So this won't run until it makes it to `main`.

Reply:
Ah got it, thanks!